crypto-ffi/bindings/js/test/bun/utils.ts

```diff
@@ -162,7 +160,11 @@ export async function invite(
     const { groupInfo, welcome } =
         await DELIVERY_SERVICE.getLatestCommitBundle();
 
-    await cc2.transaction((ctx) => ctx.processWelcomeMessage(welcome!));
+    await cc2.transaction((ctx) =>
+        ctx.processWelcomeMessage(
+            new window.ccModule.Welcome(welcome!.copyBytes())
+        )
+    );
```

Comment:
This edit doesn't obviously follow from the goals of this PR. Why did this have to change?